utils.py

```
@@ -0,0 +1,11 @@
+# simple_utils.py - A tiny utility library
+
+def reverse_string(text):
+    """Reverses the characters in a string."""
+return text[::-1]
```

Ruff (0.15.21)

[error] `return` statement outside of a function/method

(F706)